lightning/src/ln/channelmanager.rs

```diff
@@ -9050,7 +9112,37 @@ where
 		let hmac = payment_id.hmac_for_offer_payment(nonce, expanded_key);
 		let context = OffersContext::OutboundPayment { payment_id, nonce, hmac: Some(hmac) };
 		let reply_paths = self.create_blinded_paths(context)
-			.map_err(|_| Bolt12SemanticError::MissingPaths)?;
+			.map_err(|_| Bolt12CreationError::BlindedPathCreationFailed)?;
```

Comment:
We can use `Bolt12RequestError` here, that way we don't need the `From<Bolt12CreationError>` implementation.

Reply:
Oops, missed that one. Thanks!